README.md

```diff
@@ -101,51 +101,169 @@ ocm login --token=${OCM_ACCESS_TOKEN} --url=http://localhost:8000
 
 ```
 
-#### Get a new Dinosaur
-This will be empty if no Dinosaur is ever created
+#### Get a new Resource
+This will be empty if no Resource is ever created
 
 ```
 ocm get /api/maestro/v1/resources
 {
   "items": [],
-  "kind": "DinosaurList",
+  "kind": "ResourceList",
   "page": 1,
   "size": 0,
   "total": 0
 }
 ```
 
-#### Post a new Dinosaur
+#### Post a new Resource
 
 ```shell
 
 ocm post /api/maestro/v1/resources << EOF
 {
-    "species": "foo"
+  "consumer_id": "cluster1",
+  "version": 1,
+  "manifest": {
+    "apiVersion": "apps/v1",
+    "kind": "Deployment",
+    "metadata": {
+      "name": "nginx",
+      "namespace": "default"
+    },
+    "spec": {
+      "replicas": 1,
+      "selector": {
+        "matchLabels": {
+          "app": "nginx"
+        }
+      },
+      "template": {
+        "metadata": {
+          "labels": {
+            "app": "nginx"
+          }
+        },
+        "spec": {
+          "containers": [
+            {
+              "image": "nginxinc/nginx-unprivileged",
+              "name": "nginx"
+            }
+          ]
+        }
+      }
+    }
+  }
 }
 EOF
 
 ```
 
-#### Get your Dinosaur
+#### Get your Resource
 
 ```shell
 ocm get /api/maestro/v1/resources
 {
   "items": [
     {
-      "created_at":"2023-10-26T08:15:54.509653Z",
-      "href":"/api/maestro/v1/resources/2XIENcJIi9t2eBblhWVCtWLdbDZ",
-      "id":"2XIENcJIi9t2eBblhWVCtWLdbDZ",
-      "kind":"Resource",
-      "species":"foo",
-      "updated_at":"2023-10-26T08:15:54.509653Z"
+      "consumer_id": "cluster1",
+      "created_at": "2023-11-23T09:26:13.43061Z",
+      "href": "/api/maestro/v1/resources/f428e21d-71cb-47a4-8d7f-82a65d9a4048",
+      "id": "f428e21d-71cb-47a4-8d7f-82a65d9a4048",
```

Comment:
the `id` is resource id or the `id` is an identifier in the database? from the api design, i think the id should be a resource id so that we can get the resource by id. but the current code generate a ID for it `ksuid.New().String()`.

Reply:
I need to update the `ksuid` to `uuid` in following PR, I generate the sample from this PR and https://github.com/openshift-online/maestro/pull/9/commits/b0c29e452b6d3668e83e17bbe55b2f366151e302.

In https://github.com/openshift-online/maestro/pull/9/ I will update the `ksuid` to `uuid`, so that the resource id can be used as the manifestwork `metadata.name` in managed cluster.

Reply:
ID is the primary key of the table.  Because this is the Resource type already, then `id` is appropriate.

Reply:
see: https://github.com/openshift-online/maestro/pull/9/commits/b0c29e452b6d3668e83e17bbe55b2f366151e302#diff-2730c95136550051015af0087364913638868468a463babd46b988c3775d7480